src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/QuestionSteps.java

```diff
@@ -153,8 +153,8 @@ public void get_the_submitted_question() throws Throwable {
     @Given("^a standard question")
     public void aStandardQuestionRound() throws IOException{
         questionRequest = (QuestionRequest) JsonUtils.toObjectFromTestName("question/standard_question_v_0_0_5", QuestionRequest.class);
-        String onlineHearingCaseId = testContext.getScenarioContext().getCurrentOnlineHearing().getCaseId();
-        onlineHearing = onlineHearingRepository.findByCaseId(onlineHearingCaseId).get();
+        //String onlineHearingCaseId = testContext.getScenarioContext().getCurrentOnlineHearing().getCaseId();
```

Comment:
In general it is better not to commit commented out code. Feel free to deleted it as the code is already in the repository.